c7n/resources/account.py

```diff
@@ -1731,3 +1731,76 @@ def process(self, resources):
         client.put_block_public_access_configuration(
             BlockPublicAccessConfiguration=updatedConfig
         )
+
+
+@filters.register('sechub-enabled')
+class SecHubEnabled(Filter):
+    """Filter an account depending on whether security hub is enabled or not.
+
+    :example:
+
+    .. code-block:: yaml
+
+       policies:
+         - name: check-securityhub-status
+           resource: aws.account
+           filters:
+            - type: sechub-enabled
+              state: true
+
+    """
+
+    permissions = ('securityhub:DescribeHub',)
+
+    schema = type_schema('sechub-enabled', state={'type': 'boolean'})
+
+    def process(self, resources, event=None):
+        state = self.data.get('state', True)
+        client = local_session(self.manager.session_factory).client('securityhub')
+        try:
+            sechub = client.describe_hub()
+        except ClientError as e:
+            if e.response['Error']['Code'] == 'InvalidAccessException':
+                sechub = None
+            else:
+                raise
+        if state == bool(sechub):
+            return resources
+        return []
+
+
+@actions.register('set-sechub')
+class SetSechub(BaseAction):
```

Comment:
please remove this action, it doesn't represent best practices usages for a security-hub

Reply:
Dropped the action as per your suggestion

Reply:
just to clarify security hub enablement is typically a multi account orchestration to setup correctly for aggregation, its closer to something we would want to use c7n_guardian for enablement. we probably want to have the enablement for it there.

Reply:
thanks and noted. I'll take a look at c7n_guardian and see if I can add this